pkg/apis/submarinerconfig/v1alpha1/types.go

```diff
@@ -178,7 +178,7 @@ type ManagedClusterInfo struct {
 	Region string `json:"region,omitempty"`
 	// InfraId represents the infrastructure id of the managed cluster.
 	// +optional
-	InfraId string `json:"infraId,omitempty"`
+	InfraID string `json:"infraId,omitempty"`
```

Comment:
This also can't be changed IMO. It would mean makjng API changes and non-backward compatibility,

Reply:
It's OK to make Go API changes b/c it's an opt-in for users of the code to pull it in, ie they have to update their version in the go.mod and then adjust their code accordingly. However I don't think anything outside this repo is using this code anyway.